### PR TITLE
Update to migration documentation

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
User feedback on migrating from Elasticsearch 7.11+. 

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@setiah - Please review. Thanks!

File: _upgrade-to/upgrade-to.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ If your existing cluster runs an older version of Elasticsearch OSS, the first s
 
   Cluster restart upgrades work between minor versions (for example, 6.5 to 6.8) and the next major version (for example, 6.x to 7.10.2). Cluster restart upgrades are faster to perform and require fewer intermediate upgrades, but require downtime.
 
+To migrate a post-fork version of Elasticsearch (7.11+) to OpenSearch, you can use Logstash.  You'll need to employ the Elasticsearch input plugin within Logstash to extract data from the Elasticsearch cluster, and the [Logstash Output OpenSearch plugin](https://github.com/opensearch-project/logstash-output-opensearch#configuration-for-logstash-output-opensearch-plugin) to write the data to the OpenSearch 2.x cluster. We suggest using Logstash version 7.13.4 or earlier, as newer versions may encounter compatibility issues when establishing a connection with OpenSearch due to changes introduced by Elasticsearch subsequent to the fork. We strongly recommend that users test this solution with their own data to ensure effectiveness.
```
Comment (setiah) [inline _upgrade-to/upgrade-to.md:21]: lgtm! Thanks @hdhalter . @kartg can you pls confirm the logstash version mentioned is the right one?

Comment (cwillum) [inline _upgrade-to/upgrade-to.md:21]: Looks like an extra space before "You'll ...".

**Review by cwillum — APPROVED** [review]

**Review by cwillum — APPROVED** [review]
LGTM. Just an extra space.

**Pull request merged**